repositories/scheduled_executions.go

```diff
@@ -155,3 +160,78 @@ func (repo *MarbleDbRepository) UpdateScheduledExecutionStatus(
 	}
 	return true, nil
 }
+
+func (repo *MarbleDbRepository) UpdateScheduledExecution(
+	ctx context.Context,
+	exec Executor,
+	input models.UpdateScheduledExecutionInput,
+) error {
+	// uses optimistic locking based on the current status to avoid overwriting the status incorrectly
```

Comment:
I don't understand the comment regarding this function implem

Reply:
my bad, this must be leftover from a copy paste from the function above, thanks

Reply:
I accidentally corrected it in the other PR instead